### PR TITLE
fix: broken FIP links

### PR DESCRIPTION
**Pull request by vesahc**
Update to fix broken FIP links on https://fips.filecoin.io/fips due to missing .html mentioned in https://github.com/filecoin-project/FIPs/issues/559

Added some fixes for the [yaml errors](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0044.md) below as well as other formatting issues causing inconsistencies on https://fips.filecoin.io/fips.

[Preview build](https://bafybeiamy45u5tqa35cmfy6dentsj6ctmdrhl6kazgzpgcp62wjtj6hujm.on.fleek.co/)

```
7:35:11 PM 08/16/2023: YAML Exception reading /workspace/FIPS/fip-0044.md: (<unknown>): did not find expected key while parsing a block mapping at line 2 column 1
7:35:11 PM 08/16/2023: YAML Exception reading /workspace/FIPS/fip-0021.md: (<unknown>): found character that cannot start any token while scanning for the next token at line 4 column 9
7:35:11 PM 08/16/2023: YAML Exception reading /workspace/FIPS/fip-0067.md: (<unknown>): could not find expected ':' while scanning a simple key at line 8 column 1
7:35:11 PM 08/16/2023: YAML Exception reading /workspace/FRCs/frc-0042.md: (<unknown>): found character that cannot start any token while scanning for the next token at line 4 column 9
7:35:11 PM 08/16/2023: YAML Exception reading /workspace/FRCs/frc-0068.md: (<unknown>): found character that cannot start any token while scanning for the next token at line 4 column 9
7:35:11 PM 08/16/2023: YAML Exception reading /workspace/FRCs/frc-0046.md: (<unknown>): mapping values are not allowed in this context at line 3 column 6
```

### CODE REVIEW

File: FIPS/fip-0044.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ discussions-to: https://github.com/filecoin-project/FIPs/discussions/413
 status: Draft
 type: Technical (Interface)
 created: 2022-08-02
-replaces: [FIP-0035](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0035.md)
+replaces: (https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0035.md)
```
Comment (anorth) [inline FIPS/fip-0044.md:9]: Just "FIP-0035" is probably enough.

File: FIPS/fip-0067.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ author: Jakub Sztandera (@Kubuxu), Alex North (@anorth), Irene Giacomelli (@iren
 discussions-to: https://github.com/filecoin-project/FIPs/discussions/727
 status: Last Call
 type: Technical (Core)
-category Core
+Category: Core
```
Comment (anorth) [inline FIPS/fip-0067.md:8]: This looks unintentional, inconsistent with the others.

File: FRCs/frc-0042.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 fip: 0042
```
Comment (anorth) [inline FRCs/frc-0042.md:2]: Please quote this

File: FRCs/frc-0046.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-fip:0046
+fip: 0046
```
Comment (anorth) [inline FRCs/frc-0046.md:2]: Please quote this

File: FIPS/fip-0044.md (hunk 1 above)
Comment (vesahc) [reply FIPS/fip-0044.md:9]: Thanks, that was my first thought, but the only other replaces I could find was [FIP-0056](https://github.com/filecoin-project/FIPs/blob/vesahc-patch-1/FIPS/fip-0056.md) which used the link instead.

File: FIPS/fip-0067.md (hunk 2 above)
Comment (vesahc) [reply FIPS/fip-0067.md]: Fixed the case. Not sure what else to do with this as the others are somewhat inconsistent.

**Review by anorth — APPROVED** [review]
Thanks!

**Review by kaitlin-beegle — APPROVED** [review]
Thank you!!

**Pull request merged**